ocs_ci/ocs/constants.py

```diff
@@ -1422,6 +1423,23 @@
 debug_crush = 20
 """
 
+
+NOOBAA_POSTGRES_TUNING_VALUES = """
+max_connections = 300
+shared_buffers = 1GB
+effective_cache_size = 3GB
+maintenance_work_mem = 256MB
+checkpoint_completion_target = 0.9
+wal_buffers = 16MB
+default_statistics_target = 100
+random_page_cost = 1.1
+effective_io_concurrency = 300
+work_mem = 1747kB
+min_wal_size = 2GB
+max_wal_size = 8GB
+shared_preload_libraries = 'pg_stat_statements'
+"""
```

Comment:
Could we have a comment with a note explaining where it comes from?

Reply:
Added